CONTRIBUTING.md

```diff
@@ -99,19 +92,27 @@ To understand how a repository fork works, see [About forks](https://docs.github
 1. Using the terminal, go to the directory where you want to clone the Status documentation repository.
 1. Fork and clone the repository:
     
-    `gh repo fork 'status-im/help.status.im' --remote --clone=true`
+    ```sh
+    gh repo fork 'status-im/help.status.im' --remote --clone=true
+    ```
```

Comment:
Thanks for the code fence edits, @jakubgs. Not strictly required, but nice to have nonetheless.